pom.xml

```diff
@@ -137,7 +137,9 @@
 		<!-- Dependency versions -->
 		<lombok.version>1.18.12</lombok.version>
 		<h2.version>1.4.199</h2.version>
-
+        <xmlsec.version>2.1.5</xmlsec.version>
```

Comment:
I need to see whats causing this.. it all looks formatted fine in eclipse locally

Reply:
The file probably has a history of a mix of spaces and tabs.  So no matter how you format it this is going to happen in GitHub.  You probably need to replace it with latest and then redo your local changes without formatting in Eclipse.

I'll think about reformatting all the pom.xml files in a separate PR later.